CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.0.0]
+### Changed
+- **BREAKING**: Update `@metamask/eth-sig-util` to version `^6` ([#235](https://github.com/MetaMask/KeyringController/pull/235))
+  - `signPersonalMessage` now normalizes `msgParams.data` in a different way for `0` and empty strings inputs. `0` will be normalized to `0x00` and empty strings to `0x`
+- Update `@metamask/utils` package and rename dispose fn ([#234](https://github.com/MetaMask/KeyringController/pull/234))
```

Comment:
The "rename dispose function" part is a change to the functionality added in #233, so from the perspective of someone getting this update, that change didn't happen.

Perhaps we could consolidate that change with #233 and have one entry for "Destroy keyrings on reference drop"? Make no mention of "dispose" since it was never released. And we can leave this entry for the utils package update.